src/d3a/setup/config_parameter_test.py

```diff
@@ -0,0 +1,69 @@
+"""
+Copyright 2018 Grid Singularity
+This file is part of D3A.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+# from d3a.models.appliance.simple import SimpleAppliance
+from d3a.models.appliance.switchable import SwitchableAppliance
+from d3a.models.area import Area
+# from d3a.models.strategy.commercial_producer import CommercialStrategy
+# from d3a.models.strategy.finite_power_plant import FinitePowerPlant
+from d3a.models.strategy.storage import StorageStrategy
+from d3a.models.strategy.load_hours import CellTowerLoadHoursStrategy, LoadHoursStrategy
+from d3a.models.appliance.pv import PVAppliance
+from d3a.models.strategy.pv import PVStrategy
+
+
+def get_setup(config):
```

Comment:
I suggest to use a very simple setup file to save time. We dont need any trades here, do we?
Isnt there another setup file that can be reused?

Reply:
I would vouch to make this setup file as simple as possible. I don't see any other setup where both parameters are modified together.